forge-gui/res/cardsfolder/i/imperiosaur.txt

```diff
@@ -3,5 +3,5 @@ ManaCost:2 G G
 Types:Creature Dinosaur
 PT:5/5
 Text:Spend only mana produced by basic lands to cast this spell.
```

Comment:
In the Future that could be done as Static Instead I think 🤔

Reply:
Agreed, but this should be a separate MR :)